package-lock.json

```diff
@@ -769,7 +769,7 @@
     "append-field": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/append-field/-/append-field-1.0.0.tgz",
-      "integrity": "sha512-klpgFSWLW1ZEs8svjfb7g4qWY0YS5imI82dTg+QahUvJ8YqAY0P10Uk8tTyh9ZGuYEZEMaeJYCF5BFuX552hsw=="
+      "integrity": "sha1-HjRA6RXwsSA9I3SOeO3XubW0PlY="
```

Comment:
Is it intentional that this changed?

Reply:
I ran `npm install` and it modified it.

Reply:
Interesting! I wonder what causes `integrity` to change like that? 🤔

Since it's a separate change from this issue, what do you think about creating a new PR for that particular change?

Reply:
done